examples/high_level_plots.py

```diff
@@ -0,0 +1,90 @@
+"""Produce roc curves from tagger output and labels."""
+# from pathlib import Path
+
+# import h5py
+import numpy as np
+
+from puma.hlplots.results import Results
+from puma.hlplots.tagger import Tagger
+from puma.utils import get_dummy_2_taggers, logger
+
+# The line below generates dummy data which is similar to a NN output
+df = get_dummy_2_taggers(add_pt=True)
+class_ids = [0, 4, 5]
+# Remove all jets which are not trained on
+df.query(f"HadronConeExclTruthLabelID in {class_ids}", inplace=True)
+df.query("pt < 250e3", inplace=True)
+
+logger.info("Start plotting")
+
+tagger_args = {
+    "perf_var": df["pt"] / 1e3,
+    "is_light": df["HadronConeExclTruthLabelID"] == 0,
+    "is_c": df["HadronConeExclTruthLabelID"] == 4,
+    "is_b": df["HadronConeExclTruthLabelID"] == 5,
+}
+
+
+dips = Tagger("dips", template=tagger_args)
+dips.label = "dummy DIPS ($f_{c}=0.005$)"
+dips.f_c = 0.005
+dips.f_b = 0.04
```

Comment:
Why do you add f_b here? Shouldn't this be f_u?

Reply:
`f_b` is for c-tagging